packages/vue/__tests__/vfor.spec.ts

```diff
@@ -0,0 +1,39 @@
+import * as Vue from '../src'
+import { createApp, reactive, nextTick } from '../src'
+;(window as any).Vue = Vue
```

Comment:
Thanks for the feedback, I improved a little bit the test.

But let's wait for him since I have my doubts about if the test is placed correctly in the project structure. because it is a combined behavior with v-for, @click and runtime-dom.

Reply:
@znck hi, whats the status of these?

Reply:
I am not sure, let's wait for Evan to take a look at this.